core/tests/test_views.py

```diff
@@ -452,6 +452,31 @@ def test_create_community_view(client: Client, user: User) -> None:
     )
 
 
+def test_join_community_view(client: Client, user: User, community: Community) -> None:
```

Comment:
This test seems too big. 
Test should have 3 sections
- Arrange
- Action
- Assert

I see several Arrange Action and Assert sections here. 
Please split into separate unit tests with correct namings